contracts/script/deploy/mainnet/mainnet.beta.config.toml

```diff
@@ -0,0 +1,89 @@
+### CORE ###
+
+# This address gets all privileges at the end of the deployment.
+initialOwner = "0x002721B4790d97dC140a049936aA710152Ba92D5"
```

Comment:
what account is this? is this a multisig? is this your own key? is it stored in 1pass somewhere?

Reply:
DA Ops

Reply:
https://github.com/Layr-Labs/eigenda/pull/1675/commits/83dbea2135a3118e13120b300f4ee4f9a19978b0